hack/openshift_security.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+oc adm policy add-cluster-role-to-user cluster-admin -z kubevirt-infra -n kube-system
```

Comment:
We already do the cluster role binding for that account in the kubevirt.yaml. Are you sure that this is needed?

Reply:
yeah, you re right, only the two lines are actually needed now